build/yamls/flow-aggregator/base/flow-aggregator.yml

```diff
@@ -5,6 +5,80 @@ metadata:
   name: flow-aggregator
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flow-aggregator-ca
+  namespace: flow-aggregator
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: flow-aggregator-client-tls
+  namespace: flow-aggregator
+type: kubernetes.io/tls
+data:
+  tls.crt: ""
+  tls.key: ""
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: flow-aggregator-role
+  namespace: flow-aggregator
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    resourceNames: ["flow-aggregator-ca"]
+    verbs: ["get", "list", "update"]
```

Comment:
I don't think the `list` verb makes sense when specifying a resource name

Reply:
Removed. Thanks!